marklogic-data-hub/src/test/java/com/marklogic/hub/dataservices/job/DeleteJobsTest.java

```diff
@@ -0,0 +1,129 @@
+package com.marklogic.hub.dataservices.job;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.marklogic.client.DatabaseClient;
+import com.marklogic.client.FailedRequestException;
+import com.marklogic.client.dataservices.ExecCaller;
+import com.marklogic.client.dataservices.IOEndpoint;
+import com.marklogic.client.io.JacksonHandle;
+import com.marklogic.hub.AbstractHubCoreTest;
+import com.marklogic.hub.dataservices.JobService;
+import com.marklogic.hub.flow.FlowInputs;
+import com.marklogic.hub.flow.RunFlowResponse;
+import com.marklogic.hub.step.RunStepResponse;
+import com.marklogic.hub.test.Customer;
+import com.marklogic.hub.test.ReferenceModelProject;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.text.MessageFormat;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class DeleteJobsTest extends AbstractHubCoreTest {
+    protected JobService service;
+    protected ObjectMapper mapper = new ObjectMapper();
+
+    protected ObjectNode apiObj;
+    protected static String apiPath = "/ml-modules/root/data-hub/5/bulk-data-services/job/deleteJobs.api";
+
+    static class CapturingErrorListener implements ExecCaller.BulkExecCaller.ErrorListener {
+        private Throwable throwable = null;
+
+        public boolean didThrow() {
+            return this.throwable != null;
+        }
+
+        public Throwable getThrowable() {
+            return throwable;
+        }
+
+        public IOEndpoint.BulkIOEndpointCaller.ErrorDisposition processError(
+            int retryCount, Throwable throwable, IOEndpoint.CallContext callContext) {
+            this.throwable = throwable;
+            return IOEndpoint.BulkIOEndpointCaller.ErrorDisposition.STOP_ALL_CALLS;
+        }
+    }
+
+    @BeforeEach
+     void beforeEach() throws IOException {
+        Reader apiReader = new InputStreamReader(this.getClass().getResourceAsStream(apiPath));
+        apiObj = mapper.readValue(apiReader, ObjectNode.class);
+        setupAndRunFlow();
+    }
+
+    protected int getCollectionCount(DatabaseClient client, String... collections) {
```

Comment:
Should be able to use getDocCount or getJobsDocCount or getJobDocCount or getBatchDocCount